go.mod

```diff
@@ -121,6 +121,7 @@ require (
 	github.com/garyburd/redigo v1.0.1-0.20170216214944-0d253a66e6e1 // indirect
 	github.com/go-ole/go-ole v1.2.1 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/jinzhu/now v0.0.0-20180511015916-ed742868f2ae
```

Comment:
Let's not add trivial dependencies. Every dependency is a liability.

Reply:
removed the dependency